### ISSUE TITLE
Let statement name collision with variables that end in a number

### ISSUE THREAD

**Issue by Ezward** [6to5/6to5#515]
Certain naming patterns can trick the compiler into generating name collision.  In the example below, the innermost if generates code for let a1 that collides with the let a12 that is declared before the if; both let statements generate _a12 as the var name.

Here is the es6;
```
function foo() {
    var a1 = 1;
    var a12 = 12;
    if(a) {
        let a1 = 2;
        let a12 = 22;
        if(a) {
            let a1 = 3;
        }
    }
}
```

Here is the generated es5;
```
"use strict";

function foo() {
    var a1 = 1;
    var a12 = 12;
    if (a) {
        var _a1 = 2;
        var _a12 = 22;
        if (a) {
            var _a12 = 3;
        }
    }
}
```

**Comment by sebmck** [comment]
Thanks!

**Comment by Ezward** [comment]
That was fast!

**Comment by sebmck** [comment]
Fixed as of [2.13.0](https://github.com/6to5/6to5/blob/master/CHANGELOG.md#2130).
